src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -16,129 +15,87 @@
   // Constants:
   private final double EXTEND_SPEED = 1.0;
   private final double RETRACT_SPEED = -1.0;
+  private final double ROTATE_SPEED = 1.0;
 
-  private final SmartMotorController motor;
+  private final SmartMotorController extendMotor;
+  private final SmartMotorController rotateMotor;
 
-  private final Modifier validatorLimit;
-  private final ModifierGroup switchLimit;
-  private final ModifierGroup limit;
+  private final Modifier extendLimit;
+  private final Modifier rotateLimit;
 
   public Climber(ClimberMap map) {
-    Modifier upperLimit = Modifier.upperLimit(map.getUpperLimit());
-    Modifier lowerLimit = Modifier.lowerLimit(map.getLowerLimit());
-    motor = map.getMotor();
-    switchLimit = new ModifierGroup(upperLimit, lowerLimit);
-    validatorLimit = Modifier.unless(motor::errored);
-    limit = new ModifierGroup(upperLimit, lowerLimit, validatorLimit);
+
+    extendMotor = map.getExtendMotor();
+    rotateMotor = map.getRotateMotor();
+
+    extendLimit = Modifier.unless(extendMotor::errored);
+    rotateLimit = Modifier.unless(extendMotor::errored);
   }
 
   // Move the motor based off a variable speed
   public CommandBase move(DoubleSupplier speed) {
+    double nSpeed = speed.getAsDouble();
```

Comment:
This will not work the way you expect since this function is only called when mapping a command to a button.